docs/layouts/shortcodes/generated/pulsar_consumer_configuration.html

```diff
@@ -0,0 +1,180 @@
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.maxTotalReceiverQueueSizeAcrossPartitions</h5></td>
+            <td style="word-wrap: break-word;">50000</td>
+            <td>Integer</td>
+            <td>The max total receiver queue size across partitions.<br />This setting reduces the receiver queue size for individual partitions if the total receiver queue size exceeds this value.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.negativeAckRedeliveryDelayMicros</h5></td>
+            <td style="word-wrap: break-word;">60000000</td>
+            <td>Long</td>
+            <td>Delay (in μs) to wait before redelivering messages that failed to be processed.<br />When an application uses <code class="highlighter-rouge">Consumer.negativeAcknowledge(Message)</code>, failed messages are redelivered after a fixed timeout.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.poolMessages</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>Enable pooling of messages and the underlying data buffers.</td>
+        </tr>
+        <tr>
+            <td><h5>pulsar.consumer.priorityLevel</h5></td>
+            <td style="word-wrap: break-word;">0</td>
+            <td>Integer</td>
+            <td>Priority level for a consumer to which a broker gives more priority while dispatching messages in the shared subscription type.<br />The broker follows descending priorities. For example, 0=max-priority, 1, 2,...<br />In shared subscription mode, the broker first dispatches messages to the max priority level consumers if they have permits. Otherwise, the broker considers next priority level consumers.<br /><br />Example 1<br />If a subscription has consumerA with <code class="highlighter-rouge">priorityLevel</code> 0 and consumerB with <code class="highlighter-rouge">priorityLevel</code> 1, then the broker only dispatches messages to consumerA until it runs out permits and then starts dispatching messages to consumerB.<br />Example 2<br />Consumer Priority, Level, Permits
```

Comment:
```suggestion
            <td>Priority level for a consumer to which a broker gives more priorities while dispatching messages in the shared subscription type.<br />The broker follows descending priorities. For example, 0=max-priority, 1, 2,...<br />In shared subscription mode, the broker first dispatches messages to the consumers on the highest priority level if they have permits. Otherwise, the broker considers consumers on the next priority level.<br /><br />Example 1<br />If a subscription has consumer A with <code class="highlighter-rouge">priorityLevel</code> 0 and consumer B with <code class="highlighter-rouge">priorityLevel</code> 1, then the broker only dispatches messages to consumer A until it runs out permits and then starts dispatching messages to consumer B.<br />Example 2<br />Consumer Priority, Level, Permits
```